src/app/modules/ix-forms/components/ix-combobox/ix-combobox.component.spec.ts

```diff
@@ -119,4 +138,34 @@ describe('IxComboboxComponent', () => {
       expect(spectator.query('input')).toBeDisabled();
     });
   });
+
+  describe('loader', () => {
+    it('loader should be rendered if the provider receives async data', async () => {
+      spectator.setInput('provider', new SimpleAsyncComboboxProvider(of([]).pipe(delay(300))));
+      await new Promise((smth) => setTimeout(smth, 300));
+      spectator.detectChanges();
+
+      expect(spectator.query('mat-progress-spinner')).toBeVisible();
+    });
+
+    it('loader should be rendered during the loading of options after type in input', async () => {
+      const provider = [{ label: 'test1', value: 'value1' }];
+      spectator.setInput('provider', new FakeProvider(provider));
+      await new Promise((smth) => setTimeout(smth, 300));
```

Comment:
you shouldn't use set timeouts to pass time in tests, because this will actually make tests slower.
You should wrap test body in `fakeAsync` and do `tick(300)` to simulate 300ms of time.